sys-kernel/coreos-kernel/coreos-kernel-4.2.2-r10.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 2014 CoreOS, Inc.
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+COREOS_SOURCE_REVISION="-r2"
+inherit coreos-kernel
+
+DESCRIPTION="CoreOS Linux kernel"
+KEYWORDS="amd64 arm64"
+
+DEPEND="
+	app-arch/gzip
+	app-shells/bash
+	sys-apps/coreutils
+	sys-apps/findutils
+	sys-apps/grep
+	sys-apps/ignition
+	sys-apps/less
+	sys-apps/sed
+	sys-apps/shadow
+	sys-apps/systemd
+	sys-apps/util-linux
+	sys-fs/btrfs-progs
+	sys-fs/mdadm
+	sys-kernel/dracut
+	virtual/udev
+	"
+RDEPEND="${DEPEND}"
+
+# We are bad, we want to get around the sandbox.  So do the creation of the
+# cpio image in pkg_setup() where we are free to mount filesystems, chroot,
+# and other fun stuff.
+pkg_setup() {
```

Comment:
Check for `"${MERGE_TYPE}" == binary` and return. No need to run this when not building the source. Also important to not run this for binary packages when using DEPEND but not RDEPEND as noted above.